release-notes/unreleased/1793.feat.rst

```diff
@@ -0,0 +1,8 @@
+The :class:`QiskitRuntimeLocalService` was created to support a local
+testing mode. To avoid the user creating separately the class the "local"
+has been added to the valid :class:`QiskitRuntimeService` channels.
```

Comment:
```suggestion
The :class:`QiskitRuntimeLocalService` was created to support a local
testing mode. To avoid having to initialize a separate class, "local"
has been added to the valid :class:`QiskitRuntimeService` channel.
```